scripts/etch-language-tests/ml/optimiser.etch

```diff
@@ -0,0 +1,41 @@
+function main()
```

Comment:
This only checks that code involving optimisers compiles and does not crash. It would be nice to know that they also do the right thing. Add assertions, please.